data/terms.json

```diff
@@ -582,5 +582,37 @@
   {
     "term": "Fungible",
     "meaning": "Fungible is a term used to describe a token of same kind that can be transferred from one address to another."
+  },
+  {
+    "term": "ABI",
+    "meaning": "The standard way to interact with contracts in the Ethereum ecosystem, both from outside the blockchain and for contract-to-contract interaction."
```

Comment:
Some minor update here.

Kindly make it `ABI is the standard ...` not `The standard ...`